bin/zopen-download

```diff
@@ -80,7 +84,6 @@ for repo in $(echo ${repo_results}); do
     fi
     rm -f "${pax}"
     dirname=${pax%.pax.Z}
-    name=${repo%port}
```

Comment:
where is name coming from now?
can you test this with a couple 'different type' of builds if you haven't already like xxhashport ?

Reply:
I moved it up to line 57: https://github.com/ZOSOpenTools/utils/pull/38/files#diff-ae055bc1692ab016fe02d0e29a4c671e6c2d9edce53be8e72badb7774d3013baR57